### PR TITLE
 #163971960 Implementation of LFA feedback

### PR DESCRIPTION
**Pull request by sportview**
#### What does this PR do?
#163971960 implementation of feedback from the LFA

#### Descriptions of Tasks completed?
* [x]  Edit data key for parties endpoint to parties for GET all parties.
* [x]  Edit data key for party endpoint to party for GET,PATCH and DELETE methods
* [x]  Edit data key for offices endpoint to offices for GET all offices.
* [x]  Edit data key for office endpoint to an office for GET request for a single office.

#### How should this be manually tested?
1. Clone this repository
2. run the Install the requirements.txt
3. Run the application
4. Using postman, make desired requests to the respective endpoints

#### What are the relevant pivotal tracker stories?
#163971960

image

image

image

image

image

### CODE REVIEW

**Comment by Mutuba** [comment]
@sportview if you are fetching all offices or parties, the data attributes should be either `parties` or `offices`, plural, please.

**Comment by mogoria** [comment]
@sportview  looks good. Kindly change your title to match the guidelines. 
Example `#163971960 Implementation of LFA feedback `

**Comment by Mutuba** [comment]
@sportview, Kindly look at @mogoria's feedback.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 venv
 __pycache__
-.vscode
\ No newline at end of file
+.pyc
```
Comment (Mutuba) [inline .gitignore:3]: @sportview, It will not hurt to put end of line.

File: myapp/test/v1/test_party.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        self.client.post('/api/v1/parties', json=self.new_party)
+
+        res = self.client.patch('/api/v1/parties/1/Rainbow')
+        data = res.get_json()
+
+        self.assertEqual(data['status'], 200)
+        self.assertEqual(data['message'], 'party successfully updated')
+        
+
+
```
Comment (Mutuba) [inline myapp/test/v1/test_party.py]: Kindly @sportview, Put an empty trailing line

**Comment by sportview** [comment]
> @sportview looks good. Kindly change your title to match the guidelines.
> Example `#163971960 Implementation of LFA feedback `

thank you Implemented

**Comment by sportview** [comment]
> @sportview if you are fetching all offices or parties, the data attributes should be either `parties` or `offices`, plural, please.

thank you

**Pull request merged**